### PR TITLE
ADD purchase_price_recalculation module

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

**Comment by bealdav** [comment]
please @pedrobaeza

**Comment by pedrobaeza** [comment]
Hey, you abuse this wildcard, hehe. I will try to check later.

**Comment by bealdav** [comment]
Thanks

File: purchase_price_recalculation/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Allows to recompute purchase lines clicking on `Purchase recalculation` entry in purchase action button
+
+Some use cases where this module can be useful:
+
+- product tax have been updated
```
Comment (legalsylvain) [inline purchase_price_recalculation/readme/DESCRIPTION.rst:5]: Not tested, but the recomputation is not done, when tax has changed ?

**Review by legalsylvain — APPROVED** [review]
One minor change and one question.
Otherwise, LGTM. Usefull module. Thanks a lot @bealdav

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Putting as server action allows you to recompute prices on already locked or confirmed purchase orders, which is not desirable. You can prevent that showing any error on that cases.

File: purchase_price_recalculation/readme/DESCRIPTION.rst (hunk 1 above)
Comment (bealdav) [reply purchase_price_recalculation/readme/DESCRIPTION.rst:5]: tax changed on product, not on purchase line

**Comment by bealdav** [comment]
@pedro fixed as you would

**Comment by bealdav** [comment]
Please @pedrobaeza could you have a look here

**Comment by bealdav** [comment]
Please @pedrobaeza could u merge this one.
All your expectations are there.

Thanks a lot

**Review by pedrobaeza — APPROVED** [review]

**Comment by sebastienbeau** [comment]
LGTM (code review)
/ocabot merge

**Pull request merged**